api/server/router/network/backend.go

```diff
@@ -12,11 +12,11 @@ import (
 // Backend is all the methods that need to be implemented
 // to provide network specific functionality.
 type Backend interface {
-	FindNetwork(idName string) (libnetwork.Network, error)
+	FindUniqueNetwork(idName string) (libnetwork.Network, error)
```

Comment:
I wonder if we should've kept the old name here (`FindNetwork`); the issue being resolved here is a bugfix, so I don't see a real reason to rename the function (in general, I'd _expect_ a function like this to return a unique network, so not sure a change in naming is really needed)

@yongtang @vdemeester wdyt?